doc/memcached.md

```diff
@@ -0,0 +1,6 @@
+# Memcached
+
+>[!warning] 🚧 WIP
```

Comment:
Is memcached service in fact a work in progress? If not, remove this - and add some basic docs: at minimum an introduction (as to what memcached is).

#316